docs/source/releases.md

```diff
@@ -46,10 +46,10 @@ GA images aren't build from scratch but rather promoted from an existing release
 
 Support matrix table shows the version requirements for a particular **scylla-operator** version. Be sure to match these requirements, otherwise some functionality will not work.
 
-|                    | v1.7        | v1.6        | v1.5        | v1.4        | v1.3        | v1.2        | v1.1        | v1.0       |
-| :----------------: | :---------: | :---------: | :---------: | :---------: | :---------: | :---------: | :---------: | :--------: |
-| Kubernetes         | `>=1.20`    | `>=1.19.10` | `>=1.19.10` | `>=1.19.10` | `>=1.19`    | `>=1.19`    | `>=1.11`    | `>=1.11`   |
-| Scylla OS          | `>=4.3`     | `>=4.3`     | `>=4.3`     | `>=4.3`     | `>=4.2`     | `>=4.2`     | `>=4.0`     | `>=4.0`    |
-| Scylla Enterprise  | `>=2021.1`  | `>=2021.1`  | `>=2021.1`  | `>=2021.1`  | `>=2020.1`  | `>=2020.1`  | `>=2020.1`  | `>=2020.1` |
-| Scylla Manager     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`    |
-| Scylla Monitoring  | `>=3.0`     | `>=3.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`    |
+|                   |    v1.8    |       v1.7        |         v1.6         |         v1.5         |         v1.4         |       v1.3        |       v1.2        |       v1.1        |       v1.0        |
+|:-----------------:|:----------:|:-----------------:|:--------------------:|:--------------------:|:--------------------:|:-----------------:|:-----------------:|:-----------------:|:-----------------:|
+|    Kubernetes     |  `>=1.21`  | `>=1.20 && <1.25` | `>=1.19.10 && <1.25` | `>=1.19.10 && <1.25` | `>=1.19.10 && <1.25` | `>=1.19 && <1.25` | `>=1.19 && <1.25` | `>=1.11 && <1.25` | `>=1.11 && <1.25` |
```

Comment:
I'm not that sure it's true for older versions. Let's update only supported versions (1.8, 1.7, 1.6) with upper bound.

Reply:
So should 1.8 have an upper bound too?

Reply:
no, 1.8 is not yet released nor there is no kubernetes version that we know it doesn't work with

Reply:
All in all, only v1.7 and v1.6 have an upper bound.